src/main/webapp/app/guided-tour/guided-tour.service.ts

```diff
@@ -340,30 +338,6 @@ export class GuidedTourService {
         return convertedStep;
     }
 
-    /**
-     * Subscribe to guided tour settings GET request and store response value in service class variable
-     */
-    public getGuidedTourSettings() {
```

Comment:
Maybe that is a stupid question, but how does the user now get the guided tour settings?
They don't seem to be in the user when the user is loaded throught `account` and I have found no way to trigger the update method (only called on skip & finish).

Reply:
They are delivered in the json of the `account` call, but not yet saved in the user object, because some client code is missing that @lovaiible already developer in another branch and is currently integrating here.